kubernetes/operationcode_python_backend/overlays/prod/deployment.yaml

```diff
@@ -0,0 +1,15 @@
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: back-end
+spec:
+  template:
+    spec:
+      containers:
+      - name: app
+        env:
+        - name: DB_HOST
+          value: # TODO
```

Comment:
do we want to use terraform for provisioning this or do you need help setting it up?

Reply:
That's TODO because we need to wait until the last minute to snapshot the current prod before going live